src/coin_test/util/ticker.py

```diff
@@ -43,3 +43,26 @@ class AssetPair(NamedTuple):
 
     asset: Ticker
     currency: Ticker
+
+    @classmethod
```

Comment:
I feel like this should be a static method not a class method. Any stong reasoning one way or the other?

Reply:
fair I don't have a strong opinion